src/core/lib/surface/call.h

```diff
@@ -328,9 +295,10 @@ template <>
 struct ContextType<CallContext> {};
 
 // TODO(ctiller): remove once call-v3 finalized
```

Comment:
Is this TODO still correct?

Reply:
maybe? we should definitely re-evaluate once the dust settles a bit.